### ISSUE TITLE
2019-12-17 tc008/tc011 M1SS CLFTAUW(i) is missing from parameter results

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#117]
CLFTAUWi is in PARAMETERLIST and CLFTAU1 is generated and d[,mct$NORMBS] is available but CLFTAUWi is not generated in the results.

Note that parameterset == "PARAMETERLIST"

Note a new tc008 mct file has been submitted to one drive to remove non-M1SS parameters.

Parameters missing: (note AUCT, AUCT1_T2 are placeholder parameters and are "expected" to be missing from parameter results dataset). FLGACCEPT*CRIT are also expected to be missing unless appear in RETURNCOLS.

```r
> chkresults(dataset=d, mct=mct, parameterset=parameterset, results=r)
parameterset:  PARAMETERLIST 
[1] "AUCT"                 "AUCT1_T2"             "CLFTAUW"              "FLGACCEPTKELCRIT"     "FLGACCEPTPREDOSECRIT"
[6] "SPANRATIOCRIT"       
```

### parameters generated 
```r
> r
      SDEID DOSE1 DOSEC1 CMAX1 CMAXDN1 FLGACCEPTPREDOSE CMIN1 CLAST1 TMAX1 FLGACCEPTTMAX TMIN1 TLAST TLAG CTROUGH1
1 2e05758774   100  1e+08  1170   11.70                1  24.8   24.8   0.5             1    24    72    0     26.6
2 230179055   100  1e+08   981    9.81                0 216.0  216.0   0.5             1    24    72    0    245.0
3 527514060   100  1e+08   912    9.12                1  23.1   23.1   0.5             1    24    72    0     27.9
4 560495032   100  1e+08  1240   12.40                1  58.3   72.7   2.0             1     0    72    0     58.3
5 619757951   100  1e+08  1140   11.40                1  51.6   54.2   1.0             1     0    72    0     51.6
6 764973783   100  1e+08  1290   12.90                0 141.0  141.0   4.0             1    24    72    0    207.0
  CTROUGHEND1 PTROUGHR1 PTROUGHREND1        KEL KELTMLO KELTMHI KELNOPT    KELRSQ   KELRSQA FLGACCEPTKEL     THALF
1        24.8 43.984962    47.177419 0.08105121      12      72       5 0.9321508 0.9095344            1  8.551966
2       216.0  4.004082     4.541667 0.06010078      12      72       5 0.9898741 0.9864988            1 11.533082
3        23.1 32.688172    39.480519 0.08690696      12      72       5 0.9616912 0.9489217            1  7.975739
4        72.7 21.269297    17.056396 0.09440913      12      72       5 0.9678945 0.9571926            1  7.341951
5        54.2 22.093023    21.033210 0.09854452      12      72       5 0.9613381 0.9484507            1  7.033848
6       141.0  6.231884     9.148936 0.06312810      16      72       4 0.9968316 0.9952475            1 10.980010
  THALFF LASTTIME FLGACCEPTTAU    AUCALL AUCALLDN1   AUCLAST  AUCLAST1 AUCLASTC1 AUCLASTDN1    AUC1     AUC2     AUC3
1      0       72            1  7270.014  72.70014  7270.014  6984.790  6609.686   69.84790 299.150 843.1700 1757.017
2      0       72            1 17500.584 175.00584 17500.584 14147.682  9107.696  141.47682 306.500 783.6274 1712.127
3      0       72            1  6915.908  69.15908  6915.908  6657.402  6296.493   66.57402 234.975 669.1201 1460.633
4      0       72            1 11194.811 111.94811 11194.811 10562.378  9880.788  105.62378  90.075 423.0750 1558.075
5      0       72            1 10269.744 102.69744 10269.744  9807.258  9234.447   98.07258 170.400 612.9000 1673.513
6      0       72            1 15708.131 157.08131 15708.131 13625.110  9625.356  136.25110 123.000 378.2500 1221.250
      AUC4     AUC5     AUC6      AUC7      AUC8      AUC9     AUC10     AUC11 AUCINT1 AUCINT2 AUCINT3 AUCINT4 AUCINT5
1 3197.960 4304.059 5129.270  6129.222  6609.702  6984.790  7224.259  7270.014   0_0.5     0_1     0_2     0_4     0_6
2 3566.124 5379.886 7077.661  9847.766 11781.890 14147.682 16815.656 17500.584   0_0.5     0_1     0_2     0_4     0_6
3 2883.694 4022.696 4886.066  5894.480  6319.833  6657.402  6880.624  6915.908   0_0.5     0_1     0_2     0_4     0_6
4 3762.466 5498.325 6842.851  8585.845  9579.630 10562.378 11135.772 11194.811   0_0.5     0_1     0_2     0_4     0_6
5 3557.235 5129.170 6387.361  8092.196  9005.348  9807.258 10227.754 10269.744   0_0.5     0_1     0_2     0_4     0_6
6 3461.250 5738.956 7683.437 10539.159 12056.206 13625.110 15324.801 15708.131   0_0.5     0_1     0_2     0_4     0_6
  AUCINT6 AUCINT7 AUCINT8 AUCINT9 AUCINT10 AUCINT11   AUC0_12   AUC0_8  AUC13_24  AUC0_10  AUC12_30   AUCINFO
1     0_8    0_12    0_16    0_24     0_48     0_72  6129.222 5129.270  707.3260 5732.090  987.1054  7285.806
2     0_8    0_12    0_16    0_24     0_48     0_72  9847.766 7077.661 3751.6030 8583.410 5464.8940 17758.484
3     0_8    0_12    0_16    0_24     0_48     0_72  5894.480 4886.066  631.3721 5533.715  885.4480  6924.630
4     0_8    0_12    0_16    0_24     0_48     0_72  8585.845 6842.851 1688.0977 7870.489 2358.9947 11208.793
5     0_8    0_12    0_16    0_24     0_48     0_72  8092.196 6387.361 1443.7998 7401.106 2000.0576 10279.415
6     0_8    0_12    0_16    0_24     0_48     0_72 10539.159 7683.437 2626.5773 9354.106 3844.8544 15828.363
   AUCINFOC AUCINFODN   AUCINFP  AUCINFPC AUCINFPDN   AUCTAU1 AUCTAUDN1  AUMCTAU1 MRTEVIFO1 MRTEVIFP1   AUCXPCTO
1  6957.618  72.85806  7279.610  6951.422  72.79610  6984.790  69.84790  40967.07  1282.236  1948.116 0.21675686
2 13681.998 177.58484 17725.710 13649.223 177.25710 14147.682 141.47682 128949.17 44858.865 60775.357 1.45226447
3  6603.597  69.24630  6921.895  6600.862  69.21895  6657.402  66.57402  39634.84  1214.387  1863.305 0.12595578
4 10591.268 112.08793 11203.912 10586.387 112.03912 10562.378 105.62378  76425.37  8268.064 14431.698 0.12473866
5  9755.793 102.79415 10275.675  9752.053 102.75675  9807.258  98.07258  68939.15  4733.031  8203.231 0.09407886
6 12549.316 158.28363 15820.546 12541.499 158.20546 13625.110 136.25110 113138.39 25444.789 41950.857 0.75959665
    AUCXPCTP     CAV1   CLFTAU1     PTF1      PTR1  VZFTAU1 VZFTAUW1 CONC1 CONC2 CONC3 CONC4 CONC5 CONC6 CONC7 CONC8
1 0.13182534 291.0329 14.316823 3.934950 47.177419 176.6392 2858.240  26.6  1170  1010   824   626   486   347   173
2 1.27005102 589.4867  7.068296 1.297739  4.541667 117.6074 1558.746 245.0   981   928   929   925   889   810   587
3 0.08649812 277.3917 15.020875 3.204493 39.480519 172.8386 2205.981  27.9   912   826   758   667   482   385   154
4 0.08122477 440.0991  9.467565 2.685077 21.269297 100.2823 1376.559  58.3   302  1030  1240   975   769   584   315
5 0.05771744 408.6358 10.196530 2.663497 22.093023 103.4713 1655.541  51.6   630  1140   985   900   682   579   303
6 0.71056449 567.7129  7.339390 2.023910  9.148936 116.2619 2068.716 207.0   285   736   950  1290  1000   945   524
  CONC9 CONC10 CONC11 CONC12 CONCTIME1 CONCTIME2 CONCTIME3 CONCTIME4 CONCTIME5 CONCTIME6 CONCTIME7 CONCTIME8 CONCTIME9
1  79.3   24.8   2.71  1.280         0       0.5         1         2         4         6         8        12        16
2 393.0  216.0  47.40 15.500         0       0.5         1         2         4         6         8        12        16
3  69.7   23.1   2.53  0.758         0       0.5         1         2         4         6         8        12        16
4 192.0   72.7   4.12  1.320         0       0.5         1         2         4         6         8        12        16
5 167.0   54.2   2.90  0.953         0       0.5         1         2         4         6         8        12        16
6 264.0  141.0  29.00  7.590         0       0.5         1         2         4         6         8        12        16
  CONCTIME10 CONCTIME11 CONCTIME12 DI1 TAU1 TOLD1 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU  CLU     AUCU       AUMCU
1         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
2         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
3         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
4         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
5         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
6         24         48         72  NA   24     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML
  AUCNORMU AURCU CONCNORMU RATEU VOLUMEWU CLWU    STUDY SITEID   SUBJID RAND                     TREATXT TRTCD PKCOLL
1       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011004    4 PF-06700841 100 mg (PO, QD)     A  POINT
2       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011005    5 PF-06700841 100 mg (PO, QD)     A  POINT
3       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011002    2 PF-06700841 100 mg (PO, QD)     A  POINT
4       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011008    8 PF-06700841 100 mg (PO, QD)     A  POINT
5       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011001    1 PF-06700841 100 mg (PO, QD)     A  POINT
6       NA    NA  NG/ML/MG    NA       NA   NA B7931009   1001 10011007    7 PF-06700841 100 mg (PO, QD)     A  POINT
  PKBDFLD      PKTERM PERIODU PERIOD VISITU VISIT                                  ACTTRT ACTTRTC          DOSEP
1  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
2  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
3  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
4  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
5  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
6  PLASMA PF-06700841  PERIOD      1    DAY    10 PF-06700841 100mg tablet QD (as 4x25mg)       A PF-06700841 QD
  DOSPND ROUTE RACES SEX  HT    WT HTUNI WTUNI TAU1U TOLD1U                                             PKRCOM
1   DONE  ORAL ASIAN   M 162 61.80    CM    KG    HR     HR PKRCOM for Subject: 10011004 PKDATAROWID: 66605217
2   DONE  ORAL ASIAN   M 174 75.45    CM    KG    HR     HR PKRCOM for Subject: 10011005 PKDATAROWID: 66605242
3   DONE  ORAL ASIAN   M 177 78.35    CM    KG    HR     HR PKRCOM for Subject: 10011002 PKDATAROWID: 66605167
4   DONE  ORAL ASIAN   M 174 72.85    CM    KG    HR     HR PKRCOM for Subject: 10011008 PKDATAROWID: 66605317
5   DONE  ORAL ASIAN   M 173 62.50    CM    KG    HR     HR PKRCOM for Subject: 10011001 PKDATAROWID: 66605142
6   DONE  ORAL ASIAN   M 163 56.20    CM    KG    HR     HR PKRCOM for Subject: 10011007 PKDATAROWID: 66605292
```

**Comment by jhhughes256** [comment]
This behaviour is consistent across all M1SS test cases (i.e. tc008, tc011, tc012, tc071, tc109, tc114).

**Comment by tensfeldt** [comment]
This issue still exists (CLFTAUW(i) missing) with commit be5ef37

**Comment by tensfeldt** [comment]
Missing CLFTAUW(i) still missing with commit b9591d0

**Comment by jhhughes256** [comment]
This issue has been addressed as of e681d9c. Example truncated `data_out` shown below for tc011. tc008 no longer shows this issue as well. This issue has been addressed for both nominal and actual time.

```r
> results_list$data_out
      SDEID DOSE1 DOSE2 DOSEC CMAX1 CMAX2  CMAXDN1  CMAXDN2 CMIN1 CMIN2 CLAST1 CLAST2 TMAX1  TMAX2 TMIN1 TMIN2 TLAST TLAG CTROUGH1 CTROUGH2
1  70745987   300   300   300  7460  6750 24.86667 22.50000  2310  2560   2680   2560 1.500 14.000     0    24    24    0     2310     2680
2 130509330   300   300   300  8120  9540 27.06667 31.80000  1940  2200   2290   2200 1.500 13.000     0    24    24    0     1940     2290
3 246409536   300   300   300  8130 12600 27.10000 42.00000   945  2240   2240   2420 0.667 12.667     0    12    24    0      945     2240
4 249265153   300   300   300 12200 12100 40.66667 40.33333  1620  1720   2030   1720 1.000 13.000     0    24    24    0     1620     2030
  AUMCTAU2  MRTLAST MRTLAST1 MRTLAST2 MRTEVIFO1 MRTEVIFO2 MRTEVIFP1 MRTEVIFP2 AUCXPCTO AUCXPCTP     CAV1     CAV2     CLFTAU1     CLFTAU2   CLFTAUW1
1 943255.6 11.22617 5.174684 17.28658  443.1862 1767.7168  551.3700 2349.7838 18.39514 18.39784 4551.121 4547.148 0.005493152 0.005497951 0.06618255
2 942066.2 10.94530 4.921983 16.70084  224.5912  556.8441  276.3798  680.3965 16.16340 16.21277 4403.804 4700.693 0.005676910 0.005318365 0.08215500
3 972113.2 11.34247 4.914439 16.69102  224.4750  469.6765  262.9995  505.0156 18.73363 18.72103 3978.228 4853.473 0.006284205 0.005150951 0.07485652
4 865304.0 10.48215 4.600225 16.38521  140.5347  277.5242  170.1054  322.0234 10.99008 10.94572 4408.762 4400.839 0.005670526 0.005680735 0.08407007
    CLFTAUW2     PTF1      PTF2     PTR1     PTR2    VZFTAU1    VZFTAU2  VZFTAUW1  VZFTAUW2 CONC1 CONC2 CONC3 CONC4 CONC5 CONC6 CONC7 CONC8 CONC9
1 0.06624038 1.131589 0.9214566 3.229437 2.636719 0.05278211 0.05282823 0.6359291 0.6364847 2.310  2.62  6.66  5.41  7.46  6.85  5.68  4.46  3.42
2 0.07696621 1.403332 1.5614720 4.185567 4.336364 0.05450133 0.05105911 0.7887313 0.7389162 1.940  3.07  6.31  7.61  8.12  7.09  5.21  4.31  2.99
3 0.06135736 1.806080 2.1345539 8.603175 5.625000 0.06355370 0.05209282 0.7570422 0.6205220 0.945  5.03  8.13  7.63  6.77  5.71  4.24  3.68  2.88
4 0.08422143 2.399767 2.3586413 7.530864 7.034884 0.04295573 0.04303307 0.6368530 0.6379995 1.620  2.48  8.19 12.20  8.64  7.14  5.20  3.81  2.72
```
